yongshin/.env.sample

```
@@ -0,0 +1,9 @@
+PORT = YOUR PORT NUMBER SHOULE BE HERE!
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = 127.0.0.1
+TYPEORM_USERNAME = root
+TYPEORM_PASSWORD = password
+TYPEORM_DATABASE = myDB
+TYPEORM_PORT = 3306
```

```suggestion
TYPEORM_PORT = dbPort
```
3306은 mysql 고유의 port number입니다. 다른것을 중의적으로 썻다고 하더라도 3306을 그대로 작성하게 되면 rdbms중에서도 mysql을 쓴다는 것을 간접적으로 알 수 있게 됩니다.